el/pagination.php

```diff
@@ -14,7 +14,6 @@
     */
 
     'previous' => '&laquo; Προηγούμενη',
-
```

Comment:
So it's the same as Laravel 5 https://github.com/laravel/laravel/blob/master/resources/lang/en/passwords.php